lib/vscode/src/vs/server/node/server.ts

```diff
@@ -255,6 +255,7 @@ export class Vscode {
 			instantiationService.invokeFunction((accessor) => {
 				instantiationService.createInstance(LogsDataCleaner);
 
+				const commit = typeof product.commit === 'string' ? product.commit : 'unknown';
```

Comment:
We use `product.commit || "development"` elsewhere in this file so I'm thinking we should create a global `commit` and use it in both places.

Reply:
You could do that in TypeScript? TIL! I think it would be simpler to just use that expression here too, there's no need IMO to share the logic.

Reply:
It may be simpler but it's more prone to getting out of sync with other uses of `commit`. We should make it bulletproof!